example/crawl_async_example.dart

```diff
@@ -11,12 +11,18 @@ import 'package:analyzer/dart/analysis/context_locator.dart';
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:graphs/graphs.dart';
 import 'package:path/path.dart' as p;
+import 'package:pool/pool.dart';
+
+/// Limits calls to [findImports].
+final _pool = Pool(10);
```

Comment:
Better to put this in the main function? Avoid global state? Encourage best practices...etc etc

Reply:
Done

Reply:
Hmm, thinking more about this I'm a little more inclined to make it global. We tend to make instances of `Pool` objects global because they typically reflect limits which are global to the VM - like HTTP requests, or file handles...

I guess I don't feel strongly about it either way though - it's a contrived example so it doesn't hurt to have a contrived approach.